packages/react-devtools-shared/src/hooks/parseHookNames/parseSourceAndMetadata.js

```diff
@@ -84,6 +84,7 @@ const runtimeURLToMetadataCache: LRUCache<
       );
     }
 
+console.log(`runtimeURLToMetadataCache() dispose of "${runtimeSourceURL}"`);
```

Comment:
should we remove this?

Reply:
Think I already did before you commente d:)